x-pack/plugins/upgrade_assistant/kibana.jsonc

```
@@ -1,7 +1,10 @@
 {
   "type": "plugin",
   "id": "@kbn/upgrade-assistant-plugin",
-  "owner": "@elastic/kibana-management",
+  "owner": [
+    "@elastic/kibana-management",
+    "@elastic/kibana-core"
+  ],
```

We might want to keep it as core only for now to reduce the noise to the management team.

```suggestion
  "owner": [
    "@elastic/kibana-core"
  ],
```